src/naksu/mebroutines/backup/backup.go

```diff
@@ -57,14 +57,14 @@ func MakeBackup(backupPath string) error {
 	progress.TranslateAndSetMessage("Please wait, writing backup...")
 	cloneErr := makeClone(diskUUID, backupPath)
 	if cloneErr != nil {
-		return errors.New("writing backup failed")
+		mebroutines.ShowWarningMessage(fmt.Sprintf(xlate.Get("Could not back up disk %s to %s"), diskUUID, backupPath))
```

Comment:
Jos en näe väärin, niin cloneErrin palauttaminen funktiosta tässä tilanteessa johtaa kahden varoitusdialogin näyttämiseen: ensin "Could not backup disk ..." ja sitten "Backup failed: <virheilmoitus>". Kenties nätimpää jos tässä ei näytettäisi varoitusta, vaan palautettaisiin cloneErr suoraan ja annetaan sitten MakeBackupin kutsujan näyttää virhe.

Reply:
Näin tehdään.